packages/manager/apps/web-ongoing-operations/src/pages/dashboard/allDom/AllDom.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import '@/setupTests';
+import React from 'react';
+import '@testing-library/jest-dom';
+import { vi } from 'vitest';
+import { render, waitFor } from '@testing-library/react';
+import { useResourcesIcebergV6 } from '@ovh-ux/manager-react-components';
+import AllDom from '@/pages/dashboard/allDom/AllDom';
+import { taskMeAllDom, taskMeDomain } from '@/constants';
+import { serviceInfo } from '@/__mocks__/serviceInfo';
+import { useGetDomainInformation } from '@/hooks/data/query';
+import { wrapper } from '@/utils/test.provider';
+import { allDom } from '@/__mocks__/allDom';
+
+vi.mock('react-router-dom', () => ({
+  useNavigate: () => vi.fn(() => null),
+  Navigate: vi.fn(() => null),
+  useLocation: vi.fn(),
+}));
+
+vi.mock('@/data/api/web-ongoing-operations', () => ({
+  getmeTaskDnsList: vi.fn(),
+}));
+
+vi.mock('@/hooks/data/query', () => ({
+  useGetDomainInformation: vi.fn(),
+}));
+
+describe('Dns datagrid', () => {
+  it('displays loading spinner while main request are loading', async () => {
```

Comment:
```suggestion
  it('displays loading spinner while main request are loading', () => {
```
No needed if don't have await